airbyte-integrations/connectors/destination-s3/Dockerfile

```diff
@@ -41,5 +41,5 @@ RUN /bin/bash -c 'set -e && \
        echo "unknown arch" ;\
     fi'
 
-LABEL io.airbyte.version=0.3.13
-LABEL io.airbyte.name=airbyte/destination-s3
+LABEL io.airbyte.version=0.3.14
+LABEL io.airbyte.name=airbyte/destination-s3
```

Comment:
add eof here and allow maintaners edit your branch so I can run the publish command.